handbook/business-operations/README.md

```diff
@@ -631,6 +631,8 @@ This applies to anyone who gets paid by the hour, including consultants and hour
 
 > _**Note:** If a contributor uses their own time tracking process or tools, then it is OK to track the extra time spent tracking!  Contributors at Fleet are evaluated based on their results, not the number of hours they work._
 
+Every Friday, Nathan Holliday logs into Gusto to capture contractor hours logged and send them to the appropriate managers for approval.
```

Comment:
1. "sends" (typo)
2. This is not just Gusto, nor just contractors-- it's any hourly team member.
   _Please see https://github.com/fleetdm/confidential/issues/2283#issuecomment-1478767480:_
   <img width="566" alt="image" src="https://user-images.githubusercontent.com/618009/227080683-ef02e082-5d34-42a8-8c82-f22f66c85069.png">
  > ### Informing managers about hours worked
  > Every Friday at HH:MM CT, Nathan Holliday gathers hours worked for anyone who gets paid hourly by Fleet.  This includes core team members and consultants, regardless of employment classification, and regardless whether inside or outside of the United States.
  >
  > - For every hourly core team member in Gusto or Pilot.co, look up [their manager (who they report to)](https://fleetdm.com/handbook/company#org-chart). 
  > - [Consultants](#who-isnt-a-consultant) don't have a formal reporting structure or manager.  Instead, send their hours worked to the CEO, no matter who the consultant is.
  > - Send each manager a direct message in Slack by copying and pasting the following template:
  > ```
  > Here are the hours worked by your direct reports since last Saturday at midnight (YYYY-MM-DD):
  > - 🧑‍🚀 Greg Neagle: 21.25
  > - 🧑‍🚀 Mike Arpaia: 3.5
  > - 🧑‍🚀 Erik Gomez: 38
  > - 🧑‍🚀 Jesse Peterson: 15
  > - 🧑‍🚀 Whitney Champion: 14.5
  > - 🧑‍🚀 Chris Long: 8
  > - 🧑‍🚀 Nick Anderson: 0
  > - 🧑‍🚀 Prima Virani: 32
  >
  > And here are the hours worked by consultants:
  > - 💁 Bombalurina: 0
  > - 💁 Jennyanydots: 0
  > - 💁 Skimbleshanks: 19
  > - 💁 Grizabella: 0
  >
  > More info: https://fleetdm.com/handbook/business-operations#informing-managers-about-hours-worked
  > ```
 
3. This PR needs to update to the existing row in the "Rituals" table.
   _If you're going to list the content up here, which is different from what I suggested, but makes sense to me, then dedupe the content by editing the table cell to include a brief link (e.g. `[Informing managers about hours worked](#informing-managers-about-hours-worked)`) to a new sub-heading `Informing managers about hours worked`._